spec/support/evm_spec_helper.rb

```diff
@@ -39,6 +39,12 @@ def self.clear_instance_variable(instance, ivar)
     instance.instance_variable_set(ivar, nil)
   end
 
+  def self.root_tenant
```

Comment:
I feel like this should be a verb, like, create_root_tenant or something.  Or, should this be a factory?  `FactoryGirl.create(:root_tenant)`